app/views/course/assessment/submission/logs/index.json.jbuilder

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+json.log do
```

Comment:
Perhaps we do not need to nest this under log anymore

Reply:
you're right. I'm refactoring this